### ISSUE TITLE
I do not get the example running

### ISSUE THREAD

**Issue by dancesWithCycles** [react-native-mapbox-gl/maps#1293]
**Describe the bug**  
I am following the documentation for the [example](https://github.com/react-native-mapbox-gl/maps/tree/master/example), but do not get it flying.

**To Reproduce**  
```
git clone https://github.com/react-native-mapbox-gl/maps.git
cd react-native-mapbox-gl/example
emacs accesstoken
yarn install
yarn start
//open new additional terminal
yarn android
```

Example:
Please refer to the example directory in this repository.

**Expected behavior**  
I expected the example app to start.

**Actual behavior**  
The ```yarn android``` call gives me this result:
```
yarn android
yarn run v1.22.5
error Command "android" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Versions (please complete the following information):**  
 - Platform: Android
 - Platform OS: Android 7.1.2
 - Device: Fairphone 2
 - Emulator/ Simulator: no
 - Dev OS: Ubuntu 18.04
 - react-native-mapbox-gl Version:
 ```"@mapbox/react-native-mapbox-gl": {
      "version": "file:../mapbox-react-native-mapbox-gl-6.1.4.tgz",
      "integrity": "sha512-rHEeBqlgqLxWAojlEFsa3Ll+UlYbUYy4fT/kLGs936zhR/sIMNxesa8nK46Wx3GaNXpKp7hbMfaObEI6MLnPMQ==",
      "requires": {
        "@turf/helpers": "4.6.0",
        "underscore": "^1.8.3"
      },
```
 - React Native Version: 
```"@types/react-native": {
      "version": "0.64.1",
      "resolved": "https://registry.npmjs.org/@types/react-native/-/react-native-0.64.1.tgz",
      "integrity": "sha512-5VUk12LTxawpcRLbzzTtbcFVHCw8ro7a9dF2OsJSwNiMZAgziyqZbAv1Sw1TgoFlfX0rOxXv/tMMJ1r1HVZ8Og==",
      "requires": {
        "@types/react": "*"
      }
    },
```
I would appreciate any advice that points me in  the right direction. Cheers!

**Comment by ferdicus** [comment]
that's weird, are you within the `example` directory when executing `yarn android`?

**Comment by dancesWithCycles** [comment]
Yep. Here you have it crystal clear:
```
~/git/react-native-mapbox-gl/example$ yarn android
yarn run v1.22.5
error Command "android" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I am not surprised. There is no script in the `package.json` file that matches the call `android`. See:
```
"scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "build:ios": "node node_modules/react-native/local-cli/cli.js bundle --entry-file='index.ios.js' --bundle-output='./ios/RNMapboxGLExample/main.jsbundle' --dev=false --platform='ios' --assets-dest='./ios'",
    "copy:changes": "node ./scripts/watch_rngl.js",
    "pack:gl": "./scripts/npm_pack_rngl.sh",
    "clean:node:modules": "./scripts/clean_node_modules.sh",
    "preinstall": "npm run pack:gl",
    "postinstall": "node ./scripts/set_access_token.js",
    "test": "jest"
  },
```

**Comment by ferdicus** [comment]
https://github.com/react-native-mapbox-gl/maps/blob/fb36c80975879a4b7afa4e4f5f59deb1f45e5794/example/package.json#L5-L18

**Comment by dancesWithCycles** [comment]
Cheers mate! You opened my eyes. I was in the wrong repository. Do you know there is a similar repository from [nitaliano](https://github.com/nitaliano/react-native-mapbox-gl)? This is very confusing.

However, now being in the right repository I am facing another issue. This is what I just did:
```
cd example/
cp ../../react-native-mapbox-gl/example/accesstoken .
yarn install
yarn start
```
And here is the response:
```
~/git/maps/example$ yarn start
yarn run v1.22.5
$ react-native start
error Cannot find module 'metro-config/src/defaults/blacklist'
Require stack:
- /home/stefan/git/maps/example/metro.config.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/node_modules/import-fresh/index.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/dist/loaders.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/dist/createExplorer.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/dist/index.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/tools/config/readConfigFromDisk.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/tools/config/index.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/commands/install/install.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/commands/index.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/index.js
- /home/stefan/git/maps/example/node_modules/react-native/cli.js.
Error: Cannot find module 'metro-config/src/defaults/blacklist'
Require stack:
- /home/stefan/git/maps/example/metro.config.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/node_modules/import-fresh/index.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/dist/loaders.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/dist/createExplorer.js
- /home/stefan/git/maps/example/node_modules/cosmiconfig/dist/index.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/tools/config/readConfigFromDisk.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/tools/config/index.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/commands/install/install.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/commands/index.js
- /home/stefan/git/maps/example/node_modules/@react-native-community/cli/build/index.js
- /home/stefan/git/maps/example/node_modules/react-native/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/stefan/git/maps/example/metro.config.js:15:19)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
info Run CLI with --verbose flag for more details.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
I appreciate any advice that is pointing me in the right direction.

**Comment by ferdicus** [comment]
Looks like a more general `react-native` setup issue than something specific with this repo/ the example app. 

Did you previously initialize and run any other react-native app?

**Comment by dancesWithCycles** [comment]
My `react-native` setup works fine for my own project. There, I call `npm i` to install dependencies, `react-native start` to start the metro bundler and `react-native run-android` to run the app on my connected Fairphone 2.

Now, I did exactly the same in this repository WITHOUT working on any other repositories. Finally, it works, the app compiles and metro takes care of deploying. This is live how it is supposed to be!

Can it happen, that working on `one` react-native repository including metro bundler can influence `another` repository where I work in parallel with react-native and metro bundler? Cheers!

**Comment by ferdicus** [comment]
> My `react-native` setup works fine for my own project. There, I call `npm i` to install dependencies, `react-native start` to start the metro bundler and `react-native run-android` to run the app on my connected Fairphone 2.
> 
> Now, I did exactly the same in this repository WITHOUT working on any other repositories. Finally, it works, the app compiles and metro takes care of deploying. This is live how it is supposed to be!
> 
> Can it happen, that working on `one` react-native repository including metro bundler can influence `another` repository where I work in parallel with react-native and metro bundler? Cheers!

Yes, JS files are served from a dev server which always runs under `http://localhost:8081/`, if you have another running that will be used.
